apache2/apache2_util.c

```diff
@@ -339,12 +340,13 @@ static void send_waf_log(struct waf_lock* lock, apr_file_t* fd, const char* str1
     get_field_value("[data ", "]", str1, waf_data);
     get_field_value("[ver ", "]", str1, waf_ruleset_info);
     get_field_value("[unique_id ", "]", unique_id, waf_unique_id);
+    get_field_value("[timestamp ", "]", str1, waf_timestamp);
```

Comment:
Is it time or timestamp in v1?

Reply:
I'm not sure. When looking at Jarvis logs we have both time and timeStamp actually (i missed the case-sensitivity)
The timeStamp field seems mostly unpopulated so I think you're correct

Reply:
Its set to time in v1

"time": "2019-07-26T03:42:12.2398037Z"

Reply:
Please set it to time

Reply:
Z means UTC. It's truly UTC, right?

Reply:
@azhao155 do we need to format the timestamp as seen above?